### ISSUE TITLE
Windows 10 slow to start up and shut down

### ISSUE THREAD

**Issue by GriffinLX** [ChrisTitusTech/win10script#351]
After running the Essential Tweaks WIndows 10 is slow to start up and shut down. Undoing the Tweaks does not get rid of the issue only fully reinstalling Windows does! Issue first occurred around mid of May!

**Comment by rafay99-epic** [comment]
Yeah...I have noted that...The reboot and shutdown take too long on my laptop. I don't what is part of the script is causing this issue.
